app/src/ingest.py

```diff
@@ -0,0 +1,36 @@
+from enum import StrEnum
+from typing import TypedDict
+
+
+class Collection(StrEnum):
```

Comment:
Maybe call this `CmrCollection`, when I saw `collection: Collection` below I thought it was some standard library `Collection` class.